packages/devtools/lib/src/timeline/event_details.dart

```diff
@@ -41,7 +41,8 @@ class EventDetails extends CoreElement {
     // The size of the event details section will change as the splitter is
     // is moved. Observe resizing so that we can rebuild the flame chart canvas
     // as necessary.
-    final observer = html.ResizeObserver(allowInterop((entries, _) {
+    final observer =
```

Comment:
add a comment with a reference to the dart:html bug so we can clean this code up later when dart:html is fixed.
Fyi @terrylucas

Reply:
linked to https://github.com/dart-lang/html/issues/102